zephyr/modules/owntech_data_api/zephyr/public_api/DataAPI.cpp

```diff
@@ -231,7 +258,7 @@ float32_t DataAPI::convert(uint8_t adc_num, uint8_t pin_num, uint16_t raw_value)
 	uint8_t channel_num = this->getChannelNumber(adc_num, pin_num);
 	if (channel_num == 0)
 	{
-		return 0;
+		return -5000;
```

Comment:
We should never have a value hardcoded in a function. This should be defined as a constant in the header file.